INSTALL.md

```diff
@@ -0,0 +1,128 @@
+# System requirements and Installation
+
+## Node.js
+
+It is a node.js project. Therefore you need node.js installed. Get it from [http://nodejs.org](http://nodejs.org).
+
+You need a current 4.x version to run the software (0.12.x should also be OK).
```

Comment:
Currently we use Node.js 6 on the prod server. I'm not sure whether everything (i.e. dependencies) would still work with 0.12, so I'd rather shift this information (4 -> 6, 0.12 -> 4)